src/main/resources/templates/java-lang/type.ftl

```diff
@@ -36,6 +36,9 @@ import java.util.StringJoiner;
 </#list>
 public class ${className} implements java.io.Serializable<#if implements?has_content><#list implements as interface>, ${interface}<#if interface_has_next></#if></#list></#if> {
 
+    @java.io.Serial
+    private static final long serialVersionUID = 1L;
```

Comment:
welcome to create pr.
I only have one questions. 
1. The library is based on 1.8, and the annotation comes from Java 11.

So It is not appropriate to introduce this annotation directly. With configuration, placeholders can be more generic for compatibility with Java 1.8 and other languages.

Reply:
https://docs.oracle.com/en/java/javase/14/docs/api/java.base/java/io/Serial.html

Add: officially introduced in the  Java 14.

Reply:
Ah yes, I removed the annotation. It is not really needed anyway

Reply:
Hi, 
Is there any update on this issue?

Reply:
Maybe we can check the JDK version in the template. 

`<#assign MapperUtil=statics["com.kobylynskyi.graphql.codegen.java.Utils"]>`

and `<# if checkJdk />` ? @kobylynskyi

Reply:
I don't think we need that annotation anyway.
Sorry for a long delay in reply!